src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -304,6 +303,20 @@ export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
   }
 
   try {
+    const { data: mimir } = await axios.get<ThorchainMimir>(
+      `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/mimir`,
```

Comment:
suggestion: can reuse mimir query here, though given how light of a request this is, more of a nitpick than a suggestion

Reply:
keeping the axios query, but leveraging the common selector.